src/Hooks/Facades/Hooks.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Carnival\Hooks\Facades;
```

Comment:
We'll have other facades (extension, themes, etc.)
What do you think about moving `Facades` up one directory

Reply:
Yeah sure that makes sense.